src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -686,22 +686,25 @@ object DatabaseConversions extends LazyLogging {
     import tech.cryptonomic.conseil.tezos.OperationBalances._
 
     override def convert(from: Block) =
-      TezosOptics.Blocks.extractOperationsAlongWithInternalResults(from).flatMap {
-        case (group, (operations, internalResults)) =>
-          val mainOperationData = operations.map(
-            op =>
-              (from, group.hash, op).convertTo[Tables.OperationsRow] ->
-                  op.convertToA[List, Tables.BalanceUpdatesRow]
-          )
-          val internalOperationData = internalResults.map { oop =>
-            val op = oop.convertTo[Operation]
-            (from, group.hash, op)
-              .convertTo[Tables.OperationsRow]
-              .copy(internal = true, nonce = Some(oop.nonce.toString)) -> op
-              .convertToA[List, Tables.BalanceUpdatesRow]
-          }
-          mainOperationData ++ internalOperationData
-      }.toList
+      TezosOptics.Blocks
```

Comment:
Any change here?

Reply:
no, I don't think so, just `scalafmt` things 😄

Reply:
Ok, I've found the reformat was already on previous PRs, too.
Possibly rebasing on master might remove it?

In any case it seems quite irrelevant apart from being irritating 😁